config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -41,6 +41,56 @@ presubmits:
             memory: "6Gi"
         securityContext:
           privileged: true
+  - name: pull-kubernetes-node-e2e-containerd-decorate
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pull-kubernetes-node-e2e-containerd-decorate
+    always_run: false
+    optional: true
+    decorate: true
+    path_alias: k8s.io/kubernetes
```

Comment:
do we need an `extra_refs` to the kubernetes repository?

Reply:
Since it will be run on k/k itself, `extra_ref` is not required. AFAIK, `extra_ref` is used only when we need to clone another repository other than the one in which the test is run

Reply:
removed.